bin/setup

```diff
@@ -13,6 +13,9 @@ FileUtils.chdir APP_ROOT do
   # This script is idempotent, so that you can run it at anytime and get an expectable outcome.
   # Add necessary setup steps to this file.
 
+  puts '== Checking project file names =='
+  system! 'bin/rename-files'
+
```

Comment:
`bin/setup` is for setting up the project itself, not for setting up the template.
If we want a script for setting up the template itself I think it belongs elsewhere.